web/src/components/DatesPageDetail.tsx

```diff
@@ -1,54 +1,93 @@
 import styled from '@emotion/styled'
-import React, { ReactElement } from 'react'
+import React, { Fragment, ReactElement, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 
-import { MAX_DATE_RECURRENCES, MAX_DATE_RECURRENCES_COLLAPSED } from 'shared'
+import { MAX_DATE_RECURRENCES } from 'shared'
 import { DateModel } from 'shared/api'
 
-import useWindowDimensions from '../hooks/useWindowDimensions'
-import Collapsible from './Collapsible'
+import { CalendarTodayIcon, ClockIcon, ExpandIcon } from '../assets'
+import dimensions from '../constants/dimensions'
 import PageDetail from './PageDetail'
+import Button from './base/Button'
+import Icon from './base/Icon'
 
-const Identifier = styled.span`
-  font-weight: bold;
+const Container = styled.div`
+  display: grid;
+  width: fit-content;
+  gap: 8px 16px;
+
+  @media ${dimensions.mediumLargeViewport} {
+    grid-template-columns: auto auto;
+  }
+`
+
+const ContainerForThreeElements = styled(Container)`
+  @media ${dimensions.mediumLargeViewport} {
+    & > :nth-child(3) {
+      grid-column: 1 / 3;
+    }
+  }
+`
+
+const StyledButton = styled(Button)`
```

Comment:
Again, the interim button until we've finished the new design system, with Toni's agreement :)

<img width="168" height="46" alt="Image" src="https://github.com/user-attachments/assets/c0ac5371-0b26-4746-9c7c-74b3fd19da92" />